src/components/RoleList/index.js

```diff
@@ -9,10 +10,8 @@ class RoleList extends Component {
       <ul className="role-list__list">
         {roles.map((item, index) => {
           return (
-            <li className="role-list__item" key={index}>
-              <div className="role-list__role">{item.role}</div>
-              <div className="role-list__date">{item.from} — {item.until}</div>
-            </li>
+            <RoleItem roleName={item.role} roleDateFrom={item.from} roleDateuntil={item.until} index={index} key={index}/>
```

Comment:
the key is necessary where several elements are created. in this case, it's in RoleList, and not in RoleItem, where you do a map and therefore create several items. the key must be in the RoleItem instantiation, and there's no need to pass it to RoleItem through props!

also, I think there's no need either to pass so many props. you can pass {item} as a prop the same way we did in the Experience component. I'll write a code suggestion to illustrate my point!